docs/source/hardware/hardware_overview.md

```diff
@@ -41,8 +41,6 @@ header-rows: 1
   - 186 - 3094 GB
 :::
 
-If you are looking for information about GPUs, see {ref}`working-gpus`.
-
 If you are interested in more information about the different partitions on Discovery, including the number of nodes per partition, running time limits, job submission limits, and RAM limits, see {ref}`partition-names`.
```

Comment:
It seems that the partition-names page may have changed name a bit because the link is broken on the page. Could you adjust it in this PR. Thank you!!